### ISSUE TITLE
EventConsumerImpl sets AutoAccept to true

### ISSUE THREAD

**Issue by pellmann** [eclipse/hono#268]
Events should not auto-accepted and settled.

**Comment by sophokles73** [comment]
`ProtonReceiver.setAuthAccept()` 's JavaDoc reads:
```
Sets whether received deliveries should be automatically accepted (and settled) after the message handler runs for them, if no other disposition has been applied during handling.
True by default.
```
So, IMHO it is ok to set auto ack to `true`. However, we should also provide means for a developer to actively manage the disposition of received messages. This is currently not possible because `HonoClient.createEventConsumer()` currently only allows passing in a `Consumer<Message>` instance, i.e. the `Disposition` cannot be handled. This param's type should probably be changed to `BiConsumer<Disposition, Message>` or we could add another factory method accepting a `BiConsumer`.

**Comment by sophokles73** [comment]
fixed in ac6e775939d984ecddd9c31b5f92688ba822c619
